packages/cli/src/sourcegen/case.ts

```
@@ -0,0 +1,118 @@
+        continue
+      }
+
+      word += char
+      continue
+    }
+
+    if (word.length !== 0) {
+      words.push(word)
+      word = ''
+      small = false
+      continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)